README.md

```diff
@@ -30,10 +30,10 @@ Authenticating is **only** needed to access public resources requiring user cons
 
 First, you will need a Kaggle account. You can sign up [here](https://www.kaggle.com/account/login).
 
-After login, you can download your Kaggle API credentials at https://www.kaggle.com/settings by clicking on the "Create New Token" button under the "API" section.
+After login, you can download your Kaggle API credentials at https://www.kaggle.com/settings by clicking on the "Generate New Token" button under the "API" section.
 
-You have four different options to authenticate. Note that if you use `kaggle-api` (the `kaggle` command-line tool) you have
-already done Option 3 and can skip this.
+You have several options to authenticate. Note that if you use `kaggle-api` (the `kaggle` command-line tool) you have
+already configured authentication and can skip this.
 
 #### Option 1: Calling kagglehub.login()
```

Comment:
In a separate PR, should we also update the `login()` method to prompt the user for their api key rather than the legacy username & token?